editions/server/tiddlywiki.info

```diff
@@ -3,7 +3,9 @@
 	"plugins": [
 		"tiddlywiki/tiddlyweb",
 		"tiddlywiki/filesystem",
-		"tiddlywiki/highlight"
+		"tiddlywiki/highlight",
+		"tiddlywiki/markdown",
+		"cyberfoxar/fm-md"
```

Comment:
I think it would be better, to create a new temporary edition eg: "server-deferred". You and use the server edition as a starting point. It's less error-prone and easier to clean up after testing. 

-----

eg: If someone runs `tiddlywiki --init server` with your branch, they will inherit these settings.